diff --git a/node_modules/generator-jhipster/dist/generators/git/generator.js b/node_modules/generator-jhipster/dist/generators/git/generator.js
index 7b4565a..8953acd 100644
--- a/node_modules/generator-jhipster/dist/generators/git/generator.js
+++ b/node_modules/generator-jhipster/dist/generators/git/generator.js
@@ -125,7 +125,7 @@ export default class GitGenerator extends BaseGenerator {
                             const bpInfo = this.jhipsterConfig.blueprints.map(bp => `${bp.name}@${bp.version}`).join(', ');
                             commitMsg += ` with blueprints ${bpInfo}`;
                         }
-                        await git.add(['.']).commit(commitMsg, { '--allow-empty': null });
+                        await git.add(['.']).commit(commitMsg, { '--allow-empty': null, 'no-verify': null });
                         this.log.ok(`Application successfully committed to Git from ${repositoryRoot}.`);
                     }
                     catch (e) {
